code/go/0chain.net/smartcontract/zcnsc/benchmark_setup.go

```diff
@@ -36,12 +37,12 @@ func Setup(eventDb *event.EventDb, clients, publicKeys []string, balances cstate
 func addMockGlobalNode(balances cstate.StateContextI) {
 	gn := newGlobalNode()
 	gn.OwnerId = viper.GetString(benchmark.ZcnOwner)
-	gn.MinStakeAmount = state.Balance(config.SmartContractConfig.GetInt64(benchmark.ZcnMinStakeAmount))
+	gn.MinStakeAmount = currency.Coin(config.SmartContractConfig.GetInt64(benchmark.ZcnMinStakeAmount))
```

Comment:
Call currency function to do Coin conversion, so as others. Or perhaps change to GetUint64() to get value from config